common/compositor/vk/vkshim.h

```diff
@@ -16,6 +16,7 @@
 #define VKSHIM_H_
 
 #include <drm/drm_fourcc.h>
+#include <gbm.h>
```

Comment:
Is this needed outside of vkshim.cpp? Either way, It seems like this change belongs in the other patch...